concourse/tasks/deploy-to-govuk-paas.yml

```diff
@@ -82,4 +82,3 @@ run:
       cf map-route $CF_APP_NAME "$CDN_DOMAIN" --hostname www
 
       cf scale --process worker -i "$WORKER_INSTANCES" $CF_APP_NAME
-      cf scale --process pw_worker -i "$PW_WORKER_INSTANCES" $CF_APP_NAME
```

Comment:
Will we need to do anything to kill off the instances that already exist?
Thinking of how Bruce was saying that env values hang around unless you explicitly unset them!

Reply:
I think the removal of the worker process from the `Procfile` should do the job here

Reply:
I'm not sure, but I'm hoping with the Procfile entry gone the processes will just disappear.  If not I can manually poke around with the cf CLI.